bftengine/src/preprocessor/PreProcessor.cpp

```diff
@@ -324,7 +327,10 @@ void PreProcessor::onMessage<PreProcessRequestMsg>(PreProcessRequestMsg *msg) {
   const NodeIdType &clientId = preProcessReqMsg->clientId();
   LOG_DEBUG(logger(), "Received PreProcessRequestMsg" << KVLOG(reqSeqNum, senderId, clientId));
 
-  if (myReplica_.isCurrentPrimary()) return;
+  if (myReplica_.isCurrentPrimary()) {
+    LOG_WARN(logger(), "Received PreProcessRequestMsg" << KVLOG(reqSeqNum, senderId, clientId));
```

Comment:
`Received PreProcessRequestMsg, but will ignore it because I am the primary` or something in that spirit?

Reply:
Yeah, just did not complete the log :-)

Reply:
Thanks!